Directory.Build.targets

```diff
@@ -23,4 +23,31 @@
       <Private>false</Private>
     </ProjectReference>
   </ItemDefinitionGroup>
+
+  <Target Name="GeneratePkgDef"
```

Comment:
Please put this in build\Scripts\Workarounds.tgargets and add a comment why it is needed.

Reply:
done